src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3576,7 +3576,7 @@ MatterDoorLockClusterServerPreAttributeChangedCallback(const chip::app::Concrete
         break;
 
     default:
-        res = emberAfPluginDoorLockOnUnhandledAttributeChange(attributePath.mEndpointId, attributeType, size, value);
+        res = emberAfPluginDoorLockOnUnhandledAttributeChange(attributePath.mEndpointId, attributePath, attributeType, size, value);
```

Comment:
@jrhees-cae @tcarmelveilleux @woody-apple Why are we still passing the endpoint id separately, as well as in the path?